data/transport missions.txt

```diff
@@ -1975,3 +1975,42 @@ ship "Science Drone"
 		"heat dissipation" 1
 	explode "tiny explosion" 5
 	explode "small explosion" 2
+
+
+
+mission "Loan Broker Reunion"
+	name "Transfer enforcers to <planet>"
+	description "Transport 15 enforcers from <source> to <destination> by <day> to fulfill a request from the New Boston loan broker."
+	deadline
+	passengers 15
+	minor
+	to offer
+		random < 25
```

Comment:
There should probably be something other than just a random offer chance, as right now the player could talk with James, take off, reland, go to the spaceport, and immediately meet the broker again. Perhaps a combat rating requirement to ensure that the player has done some flying around first before having returned to New Boston? Or better yet, take advantage of the new [mortgage conditions](https://github.com/endless-sky/endless-sky/commit/1e1c001f396a8e268abfe0022c3be9f79d04b60c) and only offer this if the player has no outstanding mortgages (meaning that they paid their first mortgage off), or if the player's credit score is above a certain point.

Reply:
+1; zero mortgages as well as a decent combat rating and a reasonable net worth would be good additions to the random condition IMHO.